conf/hb120rs_v2.conf

```diff
@@ -0,0 +1,88 @@
+# NHC Configuration File
+#
+# Lines are in the form "<hostmask>||<check>"
+# Hostmask is a glob, /regexp/, or {noderange}
+# Comments begin with '#'
+#
+# This file was automatically generated by nhc-genconf
+# Wed May 10 08:22:17 UTC 2023
+#
+
+#######################################################################
+###
+### NHC Configuration Variables
+###
+# * || export MARK_OFFLINE=1 NHC_CHECK_ALL=0
+
+
+#######################################################################
+###
+### Filesystem checks
+###
+ * || check_fs_mount_rw -t "ext4" -s "/dev/root" -f "/"
+ * || check_fs_mount_rw -t "devtmpfs" -s "devtmpfs" -f "/dev"
+ * || check_fs_mount_rw -t "sysfs" -s "sysfs" -f "/sys"
+ * || check_fs_mount_rw -t "proc" -s "proc" -f "/proc"
+ * || check_fs_mount_rw -t "securityfs" -s "securityfs" -f "/sys/kernel/security"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/dev/shm"
+ * || check_fs_mount_rw -t "devpts" -s "devpts" -f "/dev/pts"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run"
+ * || check_fs_mount_rw -t "pstore" -s "pstore" -f "/sys/fs/pstore"
+ * || check_fs_mount_rw -t "efivarfs" -s "efivarfs" -f "/sys/firmware/efi/efivars"
+ * || check_fs_mount_rw -t "bpf" -s "bpf" -f "/sys/fs/bpf"
+ * || check_fs_mount_rw -t "hugetlbfs" -s "hugetlbfs" -f "/dev/hugepages"
+ * || check_fs_mount_rw -t "mqueue" -s "mqueue" -f "/dev/mqueue"
+ * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
+ * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/tracing"
+ * || check_fs_mount_rw -t "fusectl" -s "fusectl" -f "/sys/fs/fuse/connections"
+ * || check_fs_mount_rw -t "configfs" -s "configfs" -f "/sys/kernel/config"
+ * || check_fs_mount_rw -t "vfat" -s "/dev/sda15" -f "/boot/efi"
+ * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/debug/tracing"
+ * || check_fs_mount_rw -t "ext4" -s "/dev/sdb1" -f "/mnt"
+ * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/user/1000"
+ * || check_fs_used / 90%
+ * || check_fs_used /dev 90%
+ * || check_fs_used /boot/efi 90%
+ * || check_fs_used /mnt 90%
+ * || check_fs_iused / 100%
+ * || check_fs_iused /dev 98%
+ * || check_fs_iused /mnt 100%
+
+
+#######################################################################
+###
+### Hardware checks
+###
+ * || check_hw_cpuinfo 2 120 120
+ * || check_hw_physmem 470543164kB 470543164kB 3%
+ * || check_hw_swap 0kB 0kB 3%
+ * || check_hw_ib 40 mlx5_an0:1
```

Comment:
will it fail if the VM is not AN enabled?

Reply:
Yes it will let me fix

Reply:
addressed in new commit